modules/migration-installing-migration-operator-manually.adoc

```diff
@@ -14,57 +14,49 @@ You can install the CAM Operator manually on an {product-title} 3 source cluster
 .Prerequisites
 
 * You must have `podman` installed.
+* You must have access to the link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry].
 
 .Procedure
 
+. Log in to the Red Hat Container Registry with your Customer Portal credentials:
++
+----
+# podman login registry.redhat.io
```

Comment:
Does this really need to be run as root (with the # prompt)? The next one uses $, so I'm assuming this prompt should be $ as well.

Reply:
I was wondering that myself. It's root in the documentation that I linked to. Probably safe to change to $.